### PR TITLE
Add dinner menu

### PR DESCRIPTION
**Pull request by vnsvakanda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,6 @@
 
 **TODO**: Each person add yourself to this table; Points deducted if one person adds everyone.
 
-| Name      | Role      | Github          |
-|:----------|-----------|-----------------|
-| Your name | Your menu | @your_github_id |
+| Name       | Role        | Github          |
+|:-----------|-------------|-----------------|
+| Phinyaphak | Dinner Menu | vnsvakanda      |
```
Comment (HelloYeew) [inline README.md:14]:
```suggestion
| Phinyaphak | Dinner Menu | @vnsvakanda      |
```
You forget `@` symbol.

**Review by HelloYeew — CHANGES_REQUESTED** [review]

File: Menu.md [hunk 2]
```diff
@@ -3,15 +3,20 @@
...
-    (use proper Markdown formatting, not literal text like this)
 
 ## Lunch 
 
 
 ## Dinner
 
+| Item                               | Price    |
+|:-----------------------------------|---------:|
+| Cauliflower Bolognese              | 350 Baht |
```
Comment (peach007) [inline Menu.md:14]: change from Baht to Bt we will be the same styles

File: README.md [hunk 3]
```diff
@@ -2,13 +2,13 @@
 
 ## [Menu](Menu.md)
 
-[Breakfast](#breakfast) | [Lunch](#lunch) | [Dinner](#dinner) | [Night food](#night-food) | [Beverages](#beverages)
+[Dinner](https://github.com/HelloYeew/ku-cafe/blob/dinner-menu/Menu.md)
```
Comment (HelloYeew) [inline README.md:5]:
```suggestion
[Dinner](Menu.md#Dinner)
```
Link is wrong

**Review by HelloYeew — CHANGES_REQUESTED** [review]

**Comment by pattanan-cyber** [comment]
Very well

**Comment by patkamon** [comment]
nice.+1

**Comment by HelloYeew** [comment]
Can you add the `@` symbol? You forget to add it.
`| Phinyaphak | Dinner Menu | @vnsvakanda      |`

**Comment by vnsvakanda** [comment]
why i still can't merge

**Comment by HelloYeew** [comment]
Fix conflict first before merge.

**Review by pattanan-cyber — APPROVED** [review]

**Pull request merged**